src/libYARP_dev/src/devices/RemoteControlBoard/RemoteControlBoard.cpp

```diff
@@ -2105,13 +2044,13 @@ class yarp::dev::RemoteControlBoard :
      * @return true/false on success/failure. Failure means encoders have not been received
      * from the server or that they are not being streamed with the expected rate.
      */
-    virtual bool getMotorEncoders(double *encs) {
+    virtual bool getMotorEncoders(double *encs)
+    {
```

Comment:
Ok, having a better look at the file, there are already different styles, so feel free to disregard the other comments except for line 1619